contracts/DeprecateERC20.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.5;
+
+// ============ External Imports ============
+import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+// ============ Internal Imports ============
+import {IPartyToken} from "./interfaces/IPartyToken.sol";
+
+/*
+DeprecateERC20
+by Anna Carroll
+*/
+contract DeprecateERC20 is Initializable {
+    // ============ Immutables ============
+
+    IERC20 public immutable oldToken;
+    uint256 public immutable exchangeRate;
+
+    // ============  Public Storage ============
+
+    IPartyToken public newToken;
+    // amount of oldToken migrated; target = 25k (all crowdfund tokens)
+    uint256 public totalMigrated;
+
+    // ============  Events ============
+
+    event Migrated(address indexed owner, uint256 oldTokenAmount);
+
+    // ======== Constructor =========
+
+    constructor(address _oldToken, uint256 _exchangeRate) {
+        // set oldToken and exchangeRate
+        oldToken = IERC20(_oldToken);
+        exchangeRate = _exchangeRate;
+    }
+
+    // ======== Initializer =========
+
+    function initialize(address _newToken) external initializer {
```

Comment:
Initializable might be overkill since this isn't upgradeable, we could just `require(newToken == address(0))`